src/pages/filter/components/button/button.component.tsx

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
-import { ButtonProps } from '../../../comparison/interfaces/filter.interface';
-import './button.scss';
+import { ButtonProps } from './button.interface';
+import './button.module.scss';
```

Comment:
How do you use classes here? Shouldn't be something like this?

```JavaScript
import styles from './buttn.module.scss';

styles.class1
styles.classs2
```

Instead of 
```JavaScript
import './buttn.module.scss';

'class1'
'classs2'
```

Reply:
Here i get classes through props, and use :global in btn.module.style